app/localization/translated/be.json

```diff
@@ -286,6 +286,7 @@
   "Common.compare": "Параўнаць",
   "Common.confirm": "Пацвердзіць",
   "Common.copyToClipboard": "Скапіраваць у буфер абмену",
+  "Common.copyToClipboardLocator": "Сродак вызначэння тыпу дэфекту было паспяхова скапіявана",
```

Comment:
My suggestion is to place this message in app/src/pages/inside/projectSettingsPageContainer/content/defectTypes/defectTypesMessages.js
Because this message is only related to defectType tab.

Reply:
Done.